CODEOWNERS

```diff
@@ -0,0 +1,38 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+## These owners will be the default owners for everything in
+## the repo. Unless a later match takes precedence,
+## @global-owner1 and @global-owner2 will be requested for
+## review when someone opens a pull request.
+#*       @global-owner1 @global-owner2
+*   @millerabel @mdebarros @elnyry @NicoDuvenage
```

Comment:
I believe you may need to change these to match the owners of the repo.

Or alternatively, you can remove this line until you can confirm the individuals who should be added here.

Reply:
Will obtain the rightful owners from Sam and updated accordingly.